paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -56,4 +58,15 @@ open class PrepareResourcesTask : DefaultTask() {
         }
     throw IllegalStateException("No source sets available")
   }
+
+  private fun Project.targetSdkVersion(): String {
```

Comment:
compileSdk. targetSdk is a different concept.

Reply:
good catch.